specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-09-30/consumption.json

```diff
@@ -0,0 +1,4538 @@
+{
```

Comment:
Please ignore. This was already reviewed on a previous PR. Adding here for completeness sake so checks pass.